src/main/scala/tech/cryptonomic/conseil/tezos/JsonDecoders.scala

```diff
@@ -105,9 +106,9 @@ object JsonDecoders {
 
     /* Collects definitions to decode delegates and their contracts */
     object Delegates {
-      //reusing much of the values used in operations
       import Numbers._
       import Scripts._
+      //reusing much of the values used in operations
```

Comment:
Why moving the comment? It was referring to the import clauses 😄

Reply:
something must've gone wrong with code reformatting/merge 😄